Neos.Neos/Classes/Controller/Frontend/NodeController.php

```diff
@@ -103,7 +103,7 @@ public function showAction(NodeInterface $node = null)
             $this->response->setHeader('Cache-Control', 'no-cache');
             if ($renderingMode !== null) {
                 // Deprecated TypoScript context variable from version 2.0.
-                $this->view->assign('editPreviewMode', $renderingMode->getTypoScriptPath());
+                $this->view->assign('editPreviewMode', $renderingMode->getFusionPath());
```

Comment:
the comment says that this variable is deprecated since 2.0 - so we should remove this now, but probably in a separate PR in order to make it explicit.